events/wc_offmap_events.txt

```diff
@@ -958,4 +958,322 @@ character_event = {
+	is_triggered_only = yes
+	
+	only_playable = yes
+	
+	option = {
+		name = EVTOPTA_WCOME.500
+		trigger = {
+			OR = {
+				has_injury_trigger = yes
+				trait = maimed
+				trait = mangled
+			}
+		}
+		remove_trait = wounded
+		remove_trait = severely_injured
+		remove_trait = maimed
+		remove_trait = mangled
+		remove_trait = infection
+		hidden_effect = {
+			random_list = {
+				15 = {
+					character_event = { id = WCVOD.300 }
+					modifier = {
+						factor = 0
+						check_variable = { which = void_mutations value = 5 }
+					}
+				}
+				85 = {}
+			}
+			}
+	}
+	option = {
+		name = EVTOPTB_WCOME.500
+		trigger = {
+			OR = {
+				has_symptom_trigger = yes
+				has_disease_trigger = yes
+			}
+		}
+		#general disease clearing effects:
+		remove_disease_trait_effect = yes
+		remove_symptoms_effect = yes
+		remove_disease_flags_effect = yes
+		hidden_effect = {
+			random_list = {
+				30 = {
+					character_event = { id = WCVOD.300 }
+					modifier = {
+						factor = 0
+						check_variable = { which = void_mutations value = 5 }
+					}
+				}
+				70 = {}
+			}
+		}
+	}
+	option = {
+		name = EVTOPTC_WCOME.500
+		trigger = {
+			OR = {
+				trait = one_eyed
+				trait = one_handed 
+				trait = one_legged 
+				trait = disfigured
+				trait = inbred
+				trait = blinded
+				trait = infirm
+				trait = clubfooted
+				trait = hunchback
+				trait = dwarf
+				trait = eunuch
+				trait = harelip
+				trait = stutter
+				trait = lisp
+				is_incapable = yes
+			}
+		}
+		random_list = {
+			10 = {
+				heal_one_handed_magic_effect = yes
+				random_list = {
+					40 = {
+						add_trait = chela_hand_mutation
+						modifier = {
+							factor = 0
+							AND = {
+								trait = void_being 
+								NOT = { trait = creature_naga }
+							}
+						}
+						modifier = {
+							factor = 0
+							check_variable = { which = void_mutations value = 5 }
+						}
+						modifier = {
+							factor = 0
+							trait = chela_hand_mutation
+						}
+					}
+					40 = {
+						add_trait = tentacle_arm_mutation
+						modifier = {
+							factor = 0
+							AND = {
+								trait = void_being 
+								NOT = { trait = creature_naga }
+							}
+						}
+						modifier = {
+							factor = 0
+							check_variable = { which = void_mutations value = 5 }
+						}
+						modifier = {
+							factor = 0
+							trait = tentacle_arm_mutation
+						}
+					}
+					20 = {
+					}
+				}
+				modifier = {
+					factor = 0
+					NOT = { trait = one_handed }
+				}
+			}
+			10 = {
+				heal_one_legged_magic_effect = yes
+				modifier = {
+					factor = 0
+					NOT = { trait = one_legged }
+				}
+			}
+			10 = {
+				heal_one_eyed_magic_effect = yes
+				random = {
+					chance = 60
+					add_trait = eyes_mutation
+					modifier = {
+							factor = 0
+							AND = {
+								trait = void_being 
+								NOT = { trait = creature_naga }
+							}
+					}
+					modifier = {
+							factor = 0
+							check_variable = { which = void_mutations value = 5 }
```

Comment:
Scripted trigger plz called `has_max_mutations_or_something`.

Reply:
i didnt knew you can make scripted triggers with modifiers